base/steer/FairRootManager.cxx

```diff
@@ -71,7 +71,8 @@ FairRootManager* FairRootManager::Instance()
 {
```

Comment:
If you decide to break user code by hiding the ctor now anyways, you might consider returning a `FairRootManager&` here as well, which is a cleaner interface because it must point to a valid object which the singleton factory guarantees.
```cpp
FairRootManager& FairRootManager::Instance() {
  if (!fgInstance) fgInstance = new FairRootManager;
  return *fgInstance;
}
```